main.py

```diff
@@ -1,22 +1,35 @@
 import urllib.request
+from urllib.parse import quote
 from bs4 import BeautifulSoup
 import xlsxwriter
 
-key = input("digita seu termo")
-key = ('%20').join(key.split(' '))
-url = 'http://google.com/complete/search?output=toolbar&q={}'.format(key)
-openurl = urllib.request.urlopen(url)
-mackup = openurl.read()
-soup = BeautifulSoup(mackup, 'html.parser')
+def extract_term(term):
+    sanitized_term = quote(term)
+    url = 'http://google.com/complete/search?output=toolbar&q={}'.format(sanitized_term)
+    openurl = urllib.request.urlopen(url)
+    mackup = openurl.read()
+    soup = BeautifulSoup(mackup, 'html.parser')
+
+    return soup
+
+key = input("digita seu termo: ")
+content = extract_term(key)
 
 workbook = xlsxwriter.Workbook('teste.xlsx')
 worksheet = workbook.add_worksheet()
+bold = workbook.add_format({'bold': True})
 
-planilha = ['A1', 'A2', 'A3', 'A4', 'A5', 'A6', 'A7', 'A8', 'A9', 'A10', 'A11', 'A12', 'A13']
-row = 0
-for link in soup.find_all('suggestion'):
+header_col = 0
+for col, link in enumerate(content.find_all('suggestion')):

```

Comment:
O **enumerate** nada mais nada menos, ele gera uma tupla enumerada para os items que você passou. Por exemplo:

```python
(
    (0, marketing digital),
    (1, marketing digital curso),
    (2, marketing digital curso bh),
    (3, marketing digital pinguim),
    (4, marketing digital blabla),
)
```

Logo, quando faz o for dessa forma `for col, link` funcionará assim:

```python
col = 0
link = marketing digital

# Segunda iteração

col = 1
link = marketing digital curso

# .... até o último item

col = 10
link = marketing curso blabla
```